index.html

```diff
@@ -492,7 +494,7 @@
       <td>
       <div class="container">
         <h2>Fatima Batool</h2>
-        <p class="title">PhD student or Enrichment student</p>
+        <p class="title">PhD student or Enrichment student</p>f
```

Comment:
I think this "f" at the end of the line is just a typo, please delete that letter and then your pull request is ready to go!